apps/recovery-relay/components/WithdrawModal/index.tsx

```diff
@@ -169,26 +169,31 @@ export const WithdrawModal = () => {
                 </>
               )}
               {!!txHash && (
-                <Typography
-                  variant='body1'
-                  paragraph
-                  sx={{
-                    display: 'flex',
-                    alignItems: 'center',
-                    '& > *': {
-                      marginRight: '0.5rem',
-                    },
-                  }}
-                >
-                  <Typography variant='body1'>Transaction hash:</Typography>
-                  {asset.getExplorerUrl ? (
-                    <Link href={asset.getExplorerUrl!('tx')(txHash)} target='_blank' rel='noopener noreferrer'>
-                      {txHash}
-                    </Link>
-                  ) : (
-                    txHash
-                  )}
-                </Typography>
+                <Box>
+                  <Typography
+                    variant='body1'
+                    paragraph
+                    sx={{
+                      display: 'flex',
+                      alignItems: 'center',
+                      '& > *': {
+                        marginRight: '0.5rem',
+                      },
+                    }}
+                  >
+                    <Typography variant='body1'>Transaction Hash:</Typography>
+                    {asset.getExplorerUrl ? (
+                      <Link href={asset.getExplorerUrl!('tx')(txHash)} target='_blank' rel='noopener noreferrer'>
+                        {txHash}
+                      </Link>
+                    ) : (
+                      txHash
```

Comment:
This will just show the word "txHash"
Also, we should always have the explorer link as far as I am concerned, so no need for this check